@here/harp-mapview/lib/image/ImageCache.ts

```diff
@@ -325,6 +345,30 @@ export class ImageCache {
         });
     }
 
+    private createOnImageLoaded(
+        imageItem: ImageItem,
+        onDone: (value?: ImageItem | PromiseLike<ImageItem | undefined> | undefined) => void
```

Comment:
```suggestion
        onDone: (value: ImageItem) => void
```
Since we pass the `imageItem` that is also passed `createOnImageLoaded`

Reply:
right, done